helm/zeno/values-production.yaml

```diff
@@ -6,12 +6,26 @@ zeno:
 
   api:
     host: api.zeno.ds.io
+    resources:
+      requests:
+        cpu: "1000m"    # 2 CPU cores
+        memory: "2Gi"   # 4GB RAM
```

Comment:
The idle memory usage of the api is around 2.8 GB right now. So may be memory requests needs to be more